refinery/data_set_manager/views.py

```diff
@@ -1018,3 +1018,41 @@ def post(self, request):
                 import_file.delay(file_store_item.uuid)
 
         return HttpResponse()  # 200
+
+
+class RemoveFileFromNodeView(View):
```

Comment:
Add-files and remove-files are used to update a data-set (edit for file_uuid), should they be a patch to the data_set v2 api or the assay/files api, since we're modifying a data set or the node v2 api. To the user they are just updating a field. @scottx611x Thoughts?

Reply:
To me it would be intuitive to see this change as a patch in the v2 Node api if all we are changing is the `file_uuid` field